main.cpp

```diff
@@ -0,0 +1,96 @@
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <opencv2/opencv.hpp>
+#include "CommandLineParser.h"
+
+
+
+struct position
```

Comment:
nit: there are different coding guidelines and style, but typically structures, classes and method names start from capital. For the fields, for class they start from lower case with m_ prefix, and for struct they start from upper case as well 
```suggestion
struct Position
```